app/views/registration/steps/_contact.html.haml

```diff
@@ -12,18 +12,18 @@
         .small-12.medium-12.large-7.columns
           .row
             .small-12.columns.field
-              %label{ for: 'enterprise_contact' } {{'enterprise_contact' | t}}:
-              %input.chunky.small-12.columns{ id: 'enterprise_contact', name: 'contact', required: true, placeholder: "Contact Name", ng: { model: 'enterprise.contact' } }
+              %label{ for: 'enterprise_contact' } {{'enterprise.registration.modal.steps.contact.contact_field' | t}}:
```

Comment:
I understand this are Angular translations but did you consider using the Rails namespace convention here (`registration.steps`)? I know the key wouldn't be associated to enterprise but that would be consistent with other namespaces and would make it easy to reuse them in Rails (if we ever need to).

Reply:
I did but I valued having a namespace that is easier to understand over that possibility. I also think it's unfortunate that /registration is the route. I would have preferred a more restful approach e.g. `enterprise.new.` and then use rails convention. So, tl dr; yes but no. :)